pinc/filter_project_list.inc

```diff
@@ -157,8 +158,7 @@ class DifficultyElement extends ProjectFilterElement
             echo " checked";
         }
         echo ">\n";
-        $difficulty_options = ['beginner' => _("Beginners Only"), 'easy' => _("Easy"), 'average' => _("Average"), 'hard' => _("Hard")];
```

Comment:
Do we care that "Beginners Only" is different than "Beginner"?

Reply:
I think consistency throughout the site is better, but am interested in @srjfoo's thoughts.

Reply:
I'd go for consistency throughout the site. We don't really need to care that "Beginners Only" is different from "Beginner".

Though it's actually an interesting question. I think the only round where anything except the filter block adds "Only" is P1, where the difficulty level is prepended to the genre in the project listing. In P2, it's "MENTORS ONLY" and in P3-F2, there isn't a prefix at all. An argument could be made that the filter having "Only" in P2-F2 is misleading, and doesn't actually need to be there in P1. However that's not particularly relevant here. I still think that for now, it's probably best to be consistent.

Reply:
> However that's not particularly relevant here.

🤦‍♀️ -- okay. I hadn't looked at the testing yet. Yes, I totally agree (following my reasoning above) that it's appropriate to take the "Only" out of the filter block.